app/models/homepage_feature.rb

```diff
@@ -0,0 +1,41 @@
+class HomepageFeature < ApplicationRecord
+  belongs_to :homepage
+  has_attached_file :featured_image
+  validates_attachment_content_type :featured_image, content_type: /\Aimage\/.*\z/, if: -> { featured_image.present? }
+  # validates :section_id, presence: true, allow_nil: true
+
+  attr_accessor :delete_image
+  before_save :check_image_deletion
+
+  def self.ransackable_associations(auth_object = nil)
+    []
+  end
+
+  def self.ransackable_attributes(auth_object = nil)
+    []
+  end
```

Comment:
Since we will not have an AA page for `homepage_features` we can just remove these methods which enable filtering in the index view